examples/canjs/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+!node_modules/canjs-localstorage/can.localstorage.js
+!node_modules/canjs/can.jquery.js
+!node_modules/jquery/dist/jquery.js
+!node_modules/todomvc-common/base.js
+!node_modules/todomvc-app-css/index.css
+!node_modules/todomvc-common/base.css
```

Comment:
ok yeah, i get around the silly git behavior with git add -f but no problem, this really seems like an unfortunate reality, I did a bunch of research into the topic when I did the marionette upgrade, however I will do the verbose version now :{